prep.sh

```diff
@@ -41,7 +41,7 @@ for LINE in $LINEZ; do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=1180.v0e40e65a81e1
+version=1183.v6cbe8f5778d5
```

Comment:
picking up https://github.com/jenkinsci/plugin-compat-tester/pull/377 (not relevant here)